docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FieldAwareFactorizationMachine.cs

```diff
@@ -1,76 +1,157 @@
 ﻿using System;
+using System.Collections.Generic;
```

Comment:
This file is produced by TT. Please ignore the difference here because it's caused by a renaming.